spec/parallel_tests/cli_spec.rb

```diff
@@ -180,6 +180,12 @@ def call(*args)
         subject.run(['test', '-n', '3', '--only-group', '2', '-t', 'my_test_runner'])
       end
 
+      it "run no group when only_group uses invalid index" do
+        group_number = 99
```

Comment:
that's kind of a counter usecase since the group number should never be greater then process number ...
os --only-group 3 good enough for this test ?

-> `runs nothing when --only-group references an empty group`